src/librustc_resolve/lib.rs

```diff
@@ -3195,8 +3195,9 @@ impl<'a> Resolver<'a> {
                             // This was an attempt to use a type parameter outside
                             // its scope.
                             if record_used {
+                                debug!("rib: {:?}", rib);
                                 resolve_error(self, span,
-                                              ResolutionError::TypeParametersFromOuterFunction);
+                                              ResolutionError::TypeParametersFromOuterFunction(&span));
```

Comment:
BTW, this line is >100 chars, failing the CI.

```
[00:06:14] tidy error: /checkout/src/librustc_resolve/lib.rs:3200: line longer than 100 chars
[00:06:16] some tidy checks failed
```

Reply:
Thanks! I'll update that before finalizing the PR :) (and run tidy again to check for other errors)